### PR TITLE
Update YOLACT integration docs

### PR DESCRIPTION
**Pull request by rahul-tuli**
This PR updates YOLACT documentation in `sparseml/integrations`, The commands have been updated to use our new training flows and the setup instructions have been made simpler(cause of the latest refactor)

Big things to note:

- [X] Updates Recipes
- [X] Updates Yolact sparsification tutorial
- [X] Updates integration Readme
- [X] Remove `setup_integrations.sh` bash script (not needed anymore)

### CODE REVIEW

**Review by bfineran — APPROVED** [review]

File: integrations/dbolya-yolact/README.md [hunk 1]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
 
-This integration adds a `--recipe` argument to the 
+`sparseml.yolact.train` adds a `--recipe` argument to the 
 [`train.py` script](https://github.com/neuralmagic/yolact/blob/master/train.py).
 The argument loads an appropriate recipe while preserving the rest of the training pipeline.
 Popular recipes used with this argument are found in the [`recipes` folder](./recipes).
 Otherwise, all other arguments and functionality remain the same as the original repository.
 
+Example train command:
+(Note kindly provide correct dataset path to the utility based on your local setup)
```
Comment (KSGulin) [inline integrations/dbolya-yolact/README.md:80]: nit
```suggestion
(Provide the correct dataset path to the utility based on your local setup)
```

**Review by KSGulin — APPROVED** [review]
Commented one nit. Otherwise LGTM

File: integrations/dbolya-yolact/README.md [hunk 2]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
+The `sparseml.yolact.download` utility provides an easy to use interface to download
+the `COCO` dataset.
+
+Simply invoke:
+```bash
+sparseml.yolact.download
+```
+
+To download `COCO` test-set invoke `sparseml.yolact.download --test`; for more information
+on this utility invoke it with the `--help` option. (Kindly note the download location of the dataset)
```
Comment (dbogunowicz) [inline integrations/dbolya-yolact/README.md:65]: nit: using `invoke` twice.

File: integrations/dbolya-yolact/README.md [hunk 3]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
+
+The `sparseml.yolact.download` utility provides an easy to use interface to download
+the `COCO` dataset.
+
+Simply invoke:
+```bash
+sparseml.yolact.download
+```
+
+To download `COCO` test-set invoke `sparseml.yolact.download --test`; for more information
```
Comment (dbogunowicz) [inline integrations/dbolya-yolact/README.md:64]:
```suggestion
To download `COCO` test-set invoke `sparseml.yolact.download --test`. For more information
```

File: integrations/dbolya-yolact/README.md (hunk 2 above)
Comment (dbogunowicz) [inline integrations/dbolya-yolact/README.md:65]:
```suggestion
on this utility invoke it with the `--help` option. (Note: by default, the dataset is downloaded to the code execution directory. )
```

**Review by dbogunowicz — COMMENTED** [review]
Great job. They say that every time a doc is written/refactored, somewhere a cute chinchilla is being born.

**Comment by rahul-tuli** [comment]
> Great job. They say that every time a doc is written/refactored, somewhere a cute chinchilla is being born.

Hahahahaha. Chinchilla's are cute

File: integrations/dbolya-yolact/README.md (hunk 2 above)
Comment (rahul-tuli) [reply integrations/dbolya-yolact/README.md]: I get that, but the sentence still sounds good to me; any alternate suggestions on this @jeanniefinks ?

**Review by corey-nm — DISMISSED** [review]
lgtm!

File: integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md [hunk 4]
```diff
@@ -69,31 +67,19 @@ Otherwise, setup scripts for [COCO](https://cocodataset.org/#home) can be found
...
-    ├── layers
-    │   ├── functions
-    │   └── modules
-    ├── scripts
-    ├── utils
-    └── web
-        ├── css
-        ├── dets
-        └── scripts
+       ├── coco
```
Comment (kylesayrs) [inline integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md:78]: nit: Should this be inward indented?

**Review by kylesayrs — APPROVED** [review]

File: integrations/dbolya-yolact/README.md [hunk 5]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
+
+The `sparseml.yolact.download` utility provides an easy to use interface to download
+the `COCO` dataset.
+
+Simply invoke:
+```bash
+sparseml.yolact.download
+```
+
+To download `COCO` test-set invoke `sparseml.yolact.download --test`. For more information
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/README.md:64]:
```suggestion
To download  a test `COCO` dataset, run `sparseml.yolact.download --test`. For more information
```

File: integrations/dbolya-yolact/README.md [hunk 6]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
+on this utility invoke it with the `--help` option. (Note: by default, the dataset is downloaded to the code execution directory. )
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/README.md:65]:
```suggestion
on this utility append the `--help` option. (Note: by default, the dataset is downloaded to the code execution directory. )
```

File: integrations/dbolya-yolact/README.md [hunk 7]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
 When appropriately combined, it becomes possible to create highly sparse and accurate models.
 
-This integration adds a `--recipe` argument to the 
+`sparseml.yolact.train` adds a `--recipe` argument to the 
 [`train.py` script](https://github.com/neuralmagic/yolact/blob/master/train.py).
 The argument loads an appropriate recipe while preserving the rest of the training pipeline.
 Popular recipes used with this argument are found in the [`recipes` folder](./recipes).
 Otherwise, all other arguments and functionality remain the same as the original repository.
 
+Example train command:
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/README.md:79]:
```suggestion
Example `train` command:
```

File: integrations/dbolya-yolact/README.md [hunk 8]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
+(Provide the correct dataset path to the utility based on your local setup)
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/README.md:80]:
```suggestion
be sure to provide the correct dataset path to the utility based on your local setup:
```

File: integrations/dbolya-yolact/README.md [hunk 9]
```diff
@@ -83,22 +108,21 @@ The following table lays out the root-level files and folders along with a descr
...
 After sparsifying a model, the 
-[`export.py` script](https://github.com/neuralmagic/yolact/blob/master/export.py)
+`sparseml.yolact.export_onnx` utility
 converts the model into deployment formats such as [ONNX](https://onnx.ai/).
 The export process is modified such that the quantized and pruned models are 
 corrected and folded properly.
 
-For example, the following command can be run from within the Neural Magic's 
-`yolact` repository folder to export a trained/sparsified model's checkpoint:
+For example, the following command can be run to export a trained/sparsified yolact
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/README.md:121]:
```suggestion
For example, the following command can be run to export a trained/sparsified YOLACT
```

File: integrations/dbolya-yolact/recipes/yolact.pruned.md [hunk 10]
```diff
@@ -155,7 +155,7 @@ When running, adjust hyper-parameters based on the training environment and data
 ## Training
 
 To set up the training environment, follow the instructions on the [integration README](../README.md).
-Using the given training script from the `yolact` directory the following command can be used to launch this recipe. 
+The following command can be used to launch this recipe. 
 Adjust the script command for your GPU device setup. 
 
 YOLACT supports DataParallel. Currently, this repo only supports YOLACT models with a DarkNet53 backbone.
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/recipes/yolact.pruned.md:161]:
```suggestion
YOLACT supports DataParallel. Currently, this repo only supports YOLACT models with a Darknet-53 backbone.
```

File: integrations/dbolya-yolact/recipes/yolact.pruned_quant.md [hunk 11]
```diff
@@ -187,14 +187,18 @@ When running, adjust hyper-parameters based on the training environment and data
 ## Training
 
 To set up the training environment, follow the instructions on the [integration README](../README.md).
-Using the given training script from the `yolact` directory the following command can be used to launch this recipe. 
-Adjust the script command for your GPU device setup. 
+The following command can be used to launch this recipe. 
+Adjust the script command for your GPU device setup.
 YOLACT supports DataParallel. Currently, this repo only supports YOLACT models with a DarkNet53 backbone.
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/recipes/yolact.pruned_quant.md:192]:
```suggestion
YOLACT supports DataParallel. Currently, this repo only supports YOLACT models with a Darknet-53 backbone.
```

File: integrations/dbolya-yolact/recipes/yolact.quantized.md [hunk 12]
```diff
@@ -57,14 +57,18 @@ When running, adjust hyper-parameters based on the training environment and data
 ## Training
 
 To set up the training environment, follow the instructions on the [integration README](../README.md).
-Using the given training script from the `yolact` directory the following command can be used to launch this recipe. 
-Adjust the script command for your GPU device setup. 
+The following command can be used to launch this recipe. 
+Adjust the script command for your GPU device setup.
 YOLACT supports DataParallel. Currently, this repo only supports YOLACT models with a DarkNet53 backbone.
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/recipes/yolact.quantized.md:62]:
```suggestion
YOLACT supports DataParallel. Currently, this repo only supports YOLACT models with a Darknet-53 backbone.
```

File: integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md [hunk 13]
```diff
@@ -69,31 +67,19 @@ Otherwise, setup scripts for [COCO](https://cocodataset.org/#home) can be found
 
 1. For this tutorial, run the COCO setup script with the following command from the root of the `yolact` repository:
     ```bash
-    bash data/scripts/COCO.sh
+    sparseml.yolact.download
     ```
    
     Download and validation of the COCO dataset will begin and take around 10 minutes to finish (based on the speed of your internet connection).
     The script downloads the COCO dataset into a `coco` folder under the data directory.
     Notice that once completed, the data is ready for training with the folder structure in the following state (only directories are shown for brevity):
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md:75]:
```suggestion
Once completed, the data is ready for training with the folder structure in the following state (only directories are shown for brevity):
```

File: integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md [hunk 14]
```diff
@@ -69,31 +67,19 @@ Otherwise, setup scripts for [COCO](https://cocodataset.org/#home) can be found
 
 1. For this tutorial, run the COCO setup script with the following command from the root of the `yolact` repository:
     ```bash
-    bash data/scripts/COCO.sh
+    sparseml.yolact.download
     ```
    
     Download and validation of the COCO dataset will begin and take around 10 minutes to finish (based on the speed of your internet connection).
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md:73]:
```suggestion
Based on your internet connection, downloading and validation of the COCO dataset will take around 10 minutes to complete.
```

File: integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md [hunk 15]
```diff
@@ -105,62 +91,43 @@ Otherwise, setup scripts for [COCO](https://cocodataset.org/#home) can be found
    - Download ImageNet-pre-trained [`DarkNet-53` backbone](https://drive.google.com/file/d/17Y431j4sagFpSReuPNoFcj9h7azDTZFf/view?usp=sharing)
```
Comment (jeanniefinks) [inline integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md:91]:
```suggestion
   - Download ImageNet-pre-trained [`Darknet-53` backbone](https://drive.google.com/file/d/17Y431j4sagFpSReuPNoFcj9h7azDTZFf/view?usp=sharing)
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
Made some suggested edits @rahul-tuli!

**Review by bfineran — DISMISSED** [review]

**Review by corey-nm — APPROVED** [review]

File: integrations/dbolya-yolact/README.md [hunk 16]
```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
...
+be sure to provide the correct dataset path to the utility based on your local setup:
+
+```bash
+sparseml.yolact.train --resume \
+zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none \
+--recipe ./quant_small-recipe_yolact.yaml \
+--train_info ./data/coco/annotations/instances_train2017.json \
+--validation_info ./data/coco/annotations/instances_val2017.json \
+--train_images ./data/coco/images \
+--validation_images ./datas/coco/images
```
Comment (rahul-tuli) [inline integrations/dbolya-yolact/README.md:89]:
```suggestion
--validation_images ./data/coco/images
```

File: integrations/dbolya-yolact/recipes/yolact.pruned.md [hunk 17]
```diff
@@ -155,16 +155,20 @@ When running, adjust hyper-parameters based on the training environment and data
...
 ```
-python train.py \
+sparseml.yolact.train \
 --recipe=../recipes/yolact.pruned.md \
---resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none
+--resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none \
+--train_info ./data/coco/annotations/instances_train2017.json \
+--validation_info ./data/coco/annotations/instances_val2017.json \
+--train_images ./data/coco/images \
+--validation_images ./datas/coco/images
```
Comment (rahul-tuli) [inline integrations/dbolya-yolact/recipes/yolact.pruned.md:173]:
```suggestion
--validation_images ./data/coco/images
```

File: integrations/dbolya-yolact/recipes/yolact.pruned_quant.md [hunk 18]
```diff
@@ -187,14 +187,18 @@ When running, adjust hyper-parameters based on the training environment and data
...
 ```
-python train.py \
+sparseml.yolact.train \
 --recipe=../recipes/yolact.pruned_quant.md \
---resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none
+--resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none \
+--train_info ./data/coco/annotations/instances_train2017.json \
+--validation_info ./data/coco/annotations/instances_val2017.json \
+--train_images ./data/coco/images \
+--validation_images ./datas/coco/images
```
Comment (rahul-tuli) [inline integrations/dbolya-yolact/recipes/yolact.pruned_quant.md:203]:
```suggestion
--validation_images ./data/coco/images
```

File: integrations/dbolya-yolact/recipes/yolact.quantized.md [hunk 19]
```diff
@@ -57,14 +57,18 @@ When running, adjust hyper-parameters based on the training environment and data
...
 ```
-python train.py \
+sparseml.yolact.train \
 --recipe=../recipes/yolact.quantized.md \
---resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none
+--resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none \
+--train_info ./data/coco/annotations/instances_train2017.json \
+--validation_info ./data/coco/annotations/instances_val2017.json \
+--train_images ./data/coco/images \
+--validation_images ./datas/coco/images
```
Comment (rahul-tuli) [inline integrations/dbolya-yolact/recipes/yolact.quantized.md:73]:
```suggestion
--validation_images ./data/coco/images
```

File: integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md [hunk 20]
```diff
@@ -102,65 +88,46 @@ Otherwise, setup scripts for [COCO](https://cocodataset.org/#home) can be found
...
 
 2) Run the following command to kickstart training
 ```bash
-python train.py 
+sparseml.yolact.train --resume \
+./weights/darknet53.pth \
+--train_info ./data/coco/annotations/instances_train2017.json \
+--validation_info ./data/coco/annotations/instances_val2017.json \
+--train_images ./data/coco/images \
+--validation_images ./datas/coco/images
```
Comment (rahul-tuli) [inline integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md:111]:
```suggestion
--validation_images ./data/coco/images
```

File: integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md [hunk 21]
```diff
@@ -192,10 +158,14 @@ The table below compares these tradeoffs and shows how to run them on the COCO d
...
-python train.py \
+sparseml.yolact.train \
 --recipe=../recipes/yolact.pruned.md \
 --resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none \
---save_folder=./pruned 
+--save_folder=./pruned \
+--train_info ./data/coco/annotations/instances_train2017.json \
+--validation_info ./data/coco/annotations/instances_val2017.json \
+--train_images ./data/coco/images \
+--validation_images ./datas/coco/images
```
Comment (rahul-tuli) [inline integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md:168]:
```suggestion
--validation_images ./data/coco/images
```

**Review by bfineran — APPROVED** [review]

**Review by corey-nm — APPROVED** [review]

**Review by kylesayrs — APPROVED** [review]

**Pull request merged**